### PR TITLE
Update to Command Security Maven Plugin 1.0.18

### PR DESCRIPTION
**Pull request by arjantijms**
Allows to build on JDK 22+

### CODE REVIEW

**Review by avpinchuk — APPROVED** [review]

**Review by dmatej — APPROVED** [review]

**Comment by dmatej** [comment]
https://repo1.maven.org/maven2/org/glassfish/build/command-security-maven-plugin/ - still missing

File: nucleus/core/bootstrap/src/main/java/com/sun/enterprise/glassfish/bootstrap/osgi/OSGiFrameworkLauncher.java [hunk 1]
```diff
@@ -85,6 +91,17 @@ public void run() {
         return framework;
     }
 
+    public static class Foo implements FrameworkListener {
```
Comment (pzygielo) [inline nucleus/core/bootstrap/src/main/java/com/sun/enterprise/glassfish/bootstrap/osgi/OSGiFrameworkLauncher.java:94]: :eyes:

Comment (arjantijms) [reply nucleus/core/bootstrap/src/main/java/com/sun/enterprise/glassfish/bootstrap/osgi/OSGiFrameworkLauncher.java]: Oops

**Comment by arjantijms** [comment]
Closing this. The listener that was added for Felix pretty printing for some reason corrupts Felix, and later causes bundle "org.glassfish.main.core.glassfish" to fail finding felix framework packages.

It's not worth it at the time to find the root cause of this.

**Pull request closed without merging**